tests/cases/fourslash/getOccurrencesReadonly1.ts

```diff
@@ -0,0 +1,7 @@
+/// <reference path="fourslash.ts" />
+
+////interface I {
+////  [|readonly|] prop: string;
+////}
+
+verify.rangesAreOccurrences(false);
```

Comment:
It might be because I don't quite get what an occurrence is, but this function name seems really strange to me`verify.rangesAreOccurrences `

Reply:
Yeah, I think it's supposed to mean something like “the contents of each of these ranges is another occurrence of the contents of every other range”